### ISSUE TITLE
python3-pympress from copr not installable on fedora 41

### ISSUE THREAD

**Issue by Blaimi** [Cimbali/pympress#326]
I updated from fedora 40 to fedora 41. During the update, gnome-software already mentioned, that `python3-pympress` will be uninstalled.

**Comment by Cimbali** [comment]
I’m not sure how fedora works, but the copr repo has a valid build for rawhide from which 41 forked. Do you maybe need to update your copr repo link or list of gpg keys? I’ve also resubmitted a fedora 41 build just in case.

**Comment by Blaimi** [comment]
I did the usual update. Since the default copr-repo config has the version as a variable, this should not be a problem. When I wanted to reinstall it, it mentioned that the dependency to python=3.12 cannot be fulfilled and the fc40-package 🤷‍♂️ . I built the package from source then which worked 😉 . I can try a new installation of f41 and the upgrade-path f40->f41 with virtual machines during the weekend.
